src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -216,7 +216,10 @@ class SourcesTree extends Component {
     }
 
     if (depth === 0) {
-      return <Svg name="domain" />;
+      return <Svg name="domain" 
+       className={classnames({
```

Comment:
can we add a className to the new `img`?

Reply:
Unfortunately while we can add a className it doesn't have any effect. SVGs have to be inlined for you to be able to style them with CSS.

Reply:
we can do a mask/background color trick. we do it elsewhere

Reply:
mozilla-central uses -moz-context-properties, but that only works for Firefox's chrome. As for debugger.html, we can either use an SVG filter, then the fill property works fine from the CSS (but SVG filters have a perf-cost). The other possibly is a mask (but that instead uses background-color instead of fill).

Reply:
@jasonLaster I am not quite clear, is there any action required from me at this point?

Reply:
@svitlana-galianova yep. just add the className to the new `img`. we'll style it later